build-system/pr-check/bundle-size-module-build.js

```diff
@@ -18,7 +18,7 @@ const jobName = 'bundle-size-module-build.js';
  * Steps to run during push builds.
  */
 function pushBuildWorkflow() {
-  timedExecOrDie('amp dist --noconfig --esm');
+  timedExecOrDie('amp dist --noconfig --esm --version_override 0000000000000');
```

Comment:
wait so different amp versions caused different bundle sizes? lol

Reply:
Brotli compression could find different substrings to backreference. Having a constant `0000…` means everything will compare to same bolierplate between changes.

Reply:
Reminds me of this conversation from a couple years ago when we first adopted brotli: https://github.com/ampproject/amphtml/pull/24418#issuecomment-530156261. Glad it's being fixed here. 😃

Reply:
Oops, definitely my bad for missing that bit when I spinned off the Prod build jobs! Thanks for fixing this Justin